pkgs/by-name/qh/qhexedit2/package.nix

```diff
@@ -48,11 +53,10 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
-  passthru = {
-    updateScript = nix-update-script { };
-    # I would use testers.testVersion except for some reason it fails, even with my patches that add a --version flag
-    # TODO: Debug why testVersion reports a non-zero status code in the nix sandbox
-  };
+  versionCheckProgram = "${placeholder "out"}/bin/${finalAttrs.meta.mainProgram}";
+  doInstallCheck = false; # I don't know why this is failing
```

Comment:
If $HOME does not exist and XDG_CONFIG_HOME is unset, it will crash with a floating point exception.
Maybe writableTmpDirAsHome would help?